integrations/tensorflow/e2e/conv_test.py

```diff
@@ -98,73 +98,73 @@ def conv2d_2452x2223_valid(self, img, kernel):
     return tf.nn.conv2d(img, kernel, [1, 1, 1, 1], "VALID", name="result")
 
 
-@tf_test_utils.compile_modules(conv2d=Conv2dModule)
+@tf_test_utils.compile_module(Conv2dModule)
 class ConvTest(tf_test_utils.SavedModelTestCase):
 
   def test_id_batch_size_1(self):
     i = np.arange(20, dtype=np.float32).reshape([1, 4, 5, 1])
     k = np.ones([1, 1, 1, 1], dtype=np.float32)
-    r = self.modules.conv2d.all.conv2d_1451x1111_valid(i, k)
+    r = self.get_module().conv2d_1451x1111_valid(i, k)
```

Comment:
please add unit tests for tf_util, for example confirm that if several backends are specified all of them are compiled and compared

Reply:
I added a test to check that all backends are compiled and saved to disk by `tf_utils.compile_tf_module`. The functionality of comparing the results of all of the backends is still in `tf_test_utils`, and is tested in `tf_test_utils_test.py`. It would be nice to add a test there that all specified backends show up in the multi results, but that seems like it would be difficult with our current implementation's reliance on using decorators to compile modules.